### ISSUE TITLE
fantasypros history csv

### ISSUE THREAD

**Issue by gethubguy** [dynastyprocess/data#5]
Hey there, found my way to your git via https://www.reddit.com/r/DynastyFF/comments/gzzaa1/question_for_all_you_data_driven_folks_do_any_of/

I looked through your database file (https://github.com/DynastyProcess/data/blob/master/files/database.csv) and that's honestly pretty darn close... 

- how often is this refreshed and can we get the 2020 draft class in this data set?

- are you able to get ADP from different sites under different constraints? (aka league size/league scoring filters ... espn, mfl, yahoo, sleeper, etc???) and are they broken down by site?

- It looks like you have aggregated consensus rankings from fantasy pros... are you able to extract the individual rankings that create those overall/info from each site?

Here's what I'm trying to do-
I don't need any of the statistics, b/c i'm just building a draft board that has my own rankings but the 40-times and stuff is great .... one of the things i'd like to do is build out an aggregate ADP  using info from all of the sites and look at consensus rankings for (where possible) both redraft and dynasty. I have my own weighting system (i weight different sites differently for example) so i have my draft board sheet in the workbook - i then want to build out a data source page with this information - i would either grab the data 1x (a week before my draft) or hopefully be able to update that data over time as it changes.

Let me know if that makes sense and if what i'm looking for is possible from your data set?

**Comment by tanho63** [comment]
Heya! Tackling a few things at a time: 

- database.csv is being deprecated in favor of smaller/multiple csvs, it's just too unwieldy at the moment for my liking. [db_playerids.csv](https://github.com/DynastyProcess/data/blob/master/files/db_playerids.csv) is available already and links to values. 
- in terms of "ADP" - I don't have a handy supply of this nor do I have the bandwidth to take on the QA endeavour of making sure it's standardized/cleaned/filtered to scoring/size filters at this point. 
- re: fantasypros, all I'm doing on that front is grabbing the front-facing rankings of what's posted at dynasty-overall.php (and then dynasty-rb, dynasty-wr, ppr-rb, etc). A lot of those rankings are paywalled and I would feel uncomfortable with circumnavigating paywalled ranks and reconstructing it at this point. 

In short, I am aggregating a weekly collection of FP consensus data (for both redraft and dynasty) and would be happy to figure out how to share that in files. I have a few player-data/combine-type results kicking around as well, but I don't yet have anything beyond the ffcalculator adp data available to share.

**Comment by gethubguy** [comment]
Cool - would definitely be interested in figuring out how to grab that info from you / share .... i'm not much in the way of a programmer (i do light java / javascript / etc...) but happy to help in anyway i can.

**Comment by tanho63** [comment]
Closed, now available with commits starting July 24
